tests-functional/tests/reliability/README.MD

```diff
@@ -0,0 +1,53 @@
+# Reliability Test Suite
+
+This test suite, located under `tests-functional/tests/reliability`, is designed to evaluate the robustness and stability of the system under challenging network and runtime conditions. These tests go beyond typical functional checks and simulate real-world scenarios that stress the system's resilience.
+
+## 🧪 What We Test
+
+The suite includes tests that simulate:
+
+* **Adverse network conditions**:
+
+  * High latency
+  * Packet loss
+  * Low bandwidth
+* **Node lifecycle disruptions**:
+
+  * Runtime IP address changes
+  * Node pause/unpause behavior
+* **Network protocols**:
+
+  * IPv4 and IPv6 coverage
+* **System rate limiting**:
+
+  * Testing throttling mechanisms under stress
+
+## 🚀 How to Run
+
+Running the reliability tests is nearly identical to running functional tests see [how-to-run guide](https://github.com/status-im/status-go/blob/develop/tests-functional/README.MD#how-to-run).
```

Comment:
Is it required to run `docker-compose.test.reliability.ipv4.yml` as part of environment, as you do in the workflow?
https://github.com/status-im/status-go/blob/2a8f142fbe91d7445ff75859431a3c656dde564e/.github/workflows/test-reliability.yml#L23-L25 https://github.com/status-im/status-go/blob/2a8f142fbe91d7445ff75859431a3c656dde564e/.github/workflows/test-reliability.yml#L51-L52